sources/packages/api/src/database/entities/application.model.ts

```diff
@@ -189,6 +189,12 @@ export class Application extends RecordDataModel {
   })
   pirDeniedOtherDesc?: string;
 
+  @Column({
```

Comment:
Other new entity models added have comment, but missing here. not a blocker. but nice to have.